src/lib/analytics/apis.web.js

```diff
@@ -0,0 +1,58 @@
+import * as SentryWeb from '@sentry/browser'
+import amplitude from 'amplitude-js'
+import { pickBy, property, values, isFunction } from 'lodash'
+
+const { mt, FS, dataLayer } = window
+
+class FullStory {
+  ready = false
+
+  constructor(fsApi) {
+    const { _fs_ready } = window // eslint-disable-line camelcase
+
+    this.promise = new Promise(resolve =>
+      assign(window, {
+        _fs_ready: () => {
+          if (isFunction(_fs_ready)) {
+            _fs_ready()
+          }
+
+          this.ready = true
+          resolve()
+        },
+      }),
+    )
+
+    const proxyGetter = (target, property) => {
+      const readFrom = [target, fsApi].find(object => property in object)
+
+      if (!readFrom) {
+        return
+      }
+
+      const value = readFrom[value]
```

Comment:
I have doubt about this because it's used before it be defined, is it ok?

Reply:
of course, it should be  `const value = readFrom[property]`

if you see an obvious error/mistake - you could just fix it, it's not required to ask me in that case

Reply:
Yes, but the unused import "values" from Lodash got me doubts